.github/workflows/clippy-checks.yml

```diff
@@ -0,0 +1,39 @@
+# MIT License
+#
+# Copyright (c) 2020 Yaroslav Bolyukin
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in all
+# copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
```

Comment:
I think we don't need to keep the license here!

Reply:
I didn't write this code but copied and pasted it. The source of it is [here](https://github.com/CertainLach/jrsonnet).

One of the conditions for copies of code licensed under  [MIT](https://choosealicense.com/licenses/mit/) is to keep the copyright notice. If we were to remove it, we would be breaking the license and not giving proper credits to the author.

Reply:
Seems that you've made your homework!!
Great!! 🎉 🎉 🎉 🎉